src/sections/pages/users/user.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Avatar } from 'react-rainbow-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faUserCircle } from '@fortawesome/free-regular-svg-icons';
+
+export default function User(props) {
+    const {
+        name,
+        src,
+    } = props;
+
+    function getIcon() {
+        if (src) {
+            return null;
+        }
+        return <FontAwesomeIcon icon={faUserCircle} size="3x" />;
```

Comment:
review this, because I believe that this logic is already implemented in the Avatar and you don't need to check this